File: Arrays/largest_in_array.cpp
// https://www.geeksforgeeks.org/problems/largest-element-in-array4009/1
class Solution {
  public:
    int largest(vector<int> &arr) {
        int ans = 0;
        for(int i = 0; i < arr.size(); i++){
            ans = max(ans, arr[i]);
        }

        return ans;
    }
};

File: Arrays/largest_in_array.java
// https://www.geeksforgeeks.org/problems/largest-element-in-array4009/1
// Java uses Math class for functions like: max, min, sqrt etc...
// Eg: Math.max(a, b)


class Solution {
    public static int largest(int[] arr) {
        int ans = arr[0];

        for(int i = 0; i < arr.length; i++){
            ans = Math.max(ans, arr[i]);
        }
        return ans;
    }
}
